javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java

```diff
@@ -18,32 +18,33 @@
 
     private SchemaInspector schemaInspector;
     private SchemaManager schemaManager;
-    private Dialect dialect;
+    private final Dialect dialect;
     private final FixedSchemaFactory schemaFactory;
+    private final PolyJDBC polyJDBC;
 
     public JaversSchemaManager(Dialect dialect, FixedSchemaFactory fixedSchemaFactory, PolyJDBC polyJDBC) {
         this.dialect = dialect;
         this.schemaFactory = fixedSchemaFactory;
-        this.schemaInspector = polyJDBC.schemaInspector();
-        this.schemaManager = polyJDBC.schemaManager();
+        this.polyJDBC = polyJDBC;
```

Comment:
niestety tu nie ma transakcji jeszcze Panie,
a jak wywołasz na polyJDBC.schemaInspector() to poly odpala pod spodem sql'e do słownika tabel